kratos/tests/geometries/test_triangle_2d_3.cpp

```diff
@@ -205,10 +205,10 @@ namespace Testing {
     auto geom = GenerateIrregularTriangle2D3<Node<3>>();
```

Comment:
I believe that @pooyan-dadvand was asking for this one to use points?

Reply:
Yes, but having a look to that I found this minor issue.